src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static Random random = new Random();
+
+    public static String getRandomColor() {
+
+        return String.valueOf(Color.values()[random.nextInt(Color.values().length)]);
```

Comment:
we shouldn't use String.valueOf, remember [jv-lottery?](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery)